src/components/rich-text-editor/rich-text-editor.ts

```diff
@@ -118,9 +138,34 @@ export class MRichTextEditor extends ModulVue implements InputManagementData, In
             options.toolbarButtons.push('insertImage');
         }
 
+        options.paragraphStyles = this.manageHeaderLevels();
         return options;
     }
 
+    public manageHeaderLevels(): any {
+
+        let headersLevel: any = {};
+
+        if (this.firstHeaderLevel === this.lastHeaderLevel) {
+            // One level of header
+            headersLevel['m-u--h' + this.firstHeaderLevel] = this.i18nHeaderTitle;
+        } else if (this.lastHeaderLevel - this.firstHeaderLevel === 1) {
+            // Two levels of header
+            headersLevel['m-u--h' + this.firstHeaderLevel] = this.i18nHeaderTitle;
+            headersLevel['m-u--h' + this.lastHeaderLevel] = this.i18nHeaderSubtitle;
+        } else {
+            // Multiple levels of header
+            let levelNumber: number = 1;
+            for (let headerLevel: number = this.firstHeaderLevel; headerLevel <= this.lastHeaderLevel; headerLevel++) {
```

Comment:
Essayer de flusher le for et utiliser le for in ou for of à la place

Reply:
Si mon premier niveau de titre est 3 et mon dernier niveau de titre est 6, j'ai l'impression que ça ne fonctionnera pas.
J'ai toujours besoin d'avoir le niveau auquel je suis rendu et son "index" pcq si on reprend mon exemple, je veux avoir ceci comme Array (attention j'ai changé les classes pour les rendre unique): 
"rte-h3": "Titre 1",
"rte-h4": "Titre 2",
"rte-h5": "Titre 3",
"rte-h6": "Titre 4"